test/integration.js

```diff
@@ -740,5 +752,99 @@ tape('targetingRules: multiple rules are applied, pricingBounds are honored', as
 	t.end()
 })
 
+tape('validatorWorker: does not apply empty aggr', async function(t) {
+	const channel = getValidEthChannel()
+	channel.spec = {
+		...channel.spec,
+		targetingRules: []
+	}
+
+	// Submit a new channel; we submit it to both sentries to avoid 404 when propagating messages
+	await Promise.all([
+		fetchPost(`${leaderUrl}/channel`, dummyVals.auth.leader, channel),
+		fetchPost(`${followerUrl}/channel`, dummyVals.auth.follower, channel)
+	])
+
+	// update targetingRules
+	await fetchPost(`${leaderUrl}/channel/${channel.id}/events`, dummyVals.auth.creator, {
+		events: [
+			{
+				type: constants.eventTypes.update_targeting,
+				targetingRules: [
```

Comment:
you can set this to an empty array, so that it doesn't distract from the main purpose of the test

also, did you test what happens if we comment out the fix (`if (eventReducer.isEmpty(aggr)) { return }`)?

Reply:
yeah i did it produces the a new `Accounting` state